retrieval/discovery/ec2.go

```diff
@@ -29,10 +30,14 @@ import (
 
 const (
 	ec2Label           = model.MetaLabelPrefix + "ec2_"
+	ec2LabelAz         = ec2Label + "az"
 	ec2LabelInstanceID = ec2Label + "instance_id"
+	ec2LabelPublicDns  = ec2Label + "public_dns_name"
 	ec2LabelPublicIP   = ec2Label + "public_ip"
 	ec2LabelPrivateIP  = ec2Label + "private_ip"
+	ec2LabelSubnetId   = ec2Label + "subnet_id"
```

Comment:
There's 35 fields currently in the output of this rpc, and more will be added over time. Can we do this in a way that doesn't require us to maintain individual mappings?